lib/bambora/profile.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Bambora
+  class Profile < JSONRequest
+    def initialize(client)
+      super
+
+      @path = '/v1/profiles'
+    end
+
+    def create(card_data)
+      request(method: :post, path: @path, body: card_data)
+    end
+
+    def delete(remote_id)
```

Comment:
To better align with the Bambora terminology, call this parameter, `customer_code`. Also, could this be a named parameter so that is clear? When looking at the documentation, I had to go to the implementation to determine what the input was. A named parameter would clear up that confusion immediately.